### PR TITLE
Upgrade xk6-output compatibility to v0.42.0

### LINKED ISSUE #4: Make compatible to newer k6 version v0.42.0
I've found this plugin on the k6 output site but it was not compatible with the newest version.

I've made a proposal on how to make it work again: #3 

This could also introduce the possiblity to map / transform values from sample entries as desired in other enhancements.

### PR DESCRIPTION
**Pull request by selamanse**
- Upgrade depencies
- Introduce internal metrics entry type mapping before serialization for elastic bulk
- Add Dockerfile for containerization
- Add custom workflow to push to internal registry

Closes #4

### CODE REVIEW

**Comment by selamanse** [comment]
@danielmitterdorfer can you have a look at this change any time soon? I think it would be awesome to have this in the k6 catalog as working again.

File: pkg/esoutput/mapping.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
             "enabled": true
         },
         "properties": {
-            "Time": {
```
Comment (danielmitterdorfer) [inline pkg/esoutput/mapping.json:22]: What's the reason for changing the mapping?

**Review by danielmitterdorfer — COMMENTED** [review]
Thanks for the PR and the ping @selamanse. I did not check the repo for updated PRs so I was not aware of it. IMHO the change is fine but I think the Docker image is not strictly in scope for this PR. Can you elaborate on its value? In any case, if we want to add a Docker image, we should do this in a separate PR. Would you mind removing the Docker-related changes from this PR?

File: pkg/esoutput/mapping.json (hunk 1 above)
Comment (selamanse) [reply pkg/esoutput/mapping.json]: by introducing elasticMetricEntry I renamed the fields to be distinguishable from any other entity. This was more a debugging or cosmetic issue while developing so I reverted it as it is technically unecesssary.

**Comment by selamanse** [comment]
@danielmitterdorfer I reverted the docker related additions but I left the main build for verification purposes which would execute the `make` and then zip the binary and upload it to the workflow run for review purposes... I think it might be helpful for future PRs.

**Review by danielmitterdorfer — APPROVED** [review]
Thanks for the quick turnaround. The changes look fine to me now.

**Pull request merged**